media/css/firefox/firstrun/firstrun.scss

```
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+$break-large: 1305px;
+
+@import '../../../protocol/css/includes/lib';
+
+
+//* -------------------------------------------------------------------------- */
+// Common elements & variables
+
+html {
+    height: 100%;
+    min-height: 100vh;
+}
+
+section .mzp-l-content, div.mzp-c-hero-image, .browser-logo img {
+    display: inline-block;
+    max-width: 320px;
+    text-align: left;
+}
+
+.agreement {
+    padding-bottom: $spacing-sm;
+}
+
+.browser-logo img {
+    display: block;
+    max-width: 250px;
+    padding: $spacing-lg 0 $spacing-xl;
+}
+
+.field {
+    margin: 0;
+    padding: 0;
+}
+
+.mzp-c-button {
+    width: 100%;
+}
+
+.mzp-c-hero-desc {
+    padding-bottom: $spacing-md;
+}
+
+.mzp-c-hero-image {
+    margin-top: $spacing-sm;
+}
+
+.mzp-c-hero-title {
+    @include text-display-md;
+}
+
+.fxa-email-field {
+    width: 100%;
+}
+
+.fxa-email-form-intro, .fxa-email-form-intro strong {
```

This might be a good place for some nesting... we don't want to make the whole intro `display:inline`, just the strong.
```suggestion
.fxa-email-form-intro {
     @include text-display-xs;
     font-weight: bold;
     
     strong {
         display: inline;
         font-weight: bold;
     }
}
```

---

Sigh... seems we do need to explicitly declare the font-size for the strong to override the base styling, so nesting the rules doesn't save us much repetition after all.

---

Also just noticed the mockup has this headline set in Metropolis but in the macro it's a paragraph and not a heading, so it's not getting the heading font. So... we can either try to match the mockup with a bunch of extra CSS to make a paragraph look like a heading, or just live with it as-is (though really we should change it to a heading in the macro anyway but that might cause more problems on other pages).

---

I did try changing it to a heading and I would have to update css in all the other pages that use this macro which seemed like a bigger rabbit whole then it was worth, but I can do that if needed. I can try adding some custom styles to make it closer but I was trying to stick as close to using all protocol options as possible. @craigcook

---

I vote for leaving it as-is. We can refactor the macro another time.

---

sounds good 🧟‍♀️